src/modules/producer.js

```diff
@@ -50,28 +75,31 @@ function createRpcQueue(queue) {
  */
 function maybeAnswer(queue) {
   var rpcQueue = amqpRPCQueues[queue];
-
+  
   return (msg) => {
     //check the correlation ID sent by the initial message using RPC
     var corrId = msg.properties.correlationId;
-
+    
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.dialonce.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)